tests/OAuth2AttendeesApiTest.php

```diff
@@ -240,6 +240,43 @@ public function testUpdateAttendee($summit_id = 27, $attendee_id = 18502){
         return $attendee;
     }
 
+    public function testUpdateAttendeeNotesUnicode($summit_id = 3109, $attendee_id = 1){
+        $params = [
+            'id' => $summit_id,
+            'attendee_id' => $attendee_id,
+            'expand'   => 'admin_notes'
+        ];
+
+        $data = [
+            'first_name' => 'Clint',
+            'surname' => 'Espinoza',
+            'email' => 'clint@gmail.com',
+            'admin_notes' => '嘗試特殊字符',
+        ];
+
+        $headers = [
+            "HTTP_Authorization" => " Bearer " . $this->access_token,
+            "CONTENT_TYPE"        => "application/json"
+        ];
+
+        $response = $this->action(
+            "PUT",
+            "OAuth2SummitAttendeesApiController@updateAttendee",
+            $params,
+            [],
+            [],
+            [],
+            $headers,
+            json_encode($data)
+        );
+
+        $content = $response->getContent();
+        $this->assertResponseStatus(201);
+        $attendee = json_decode($content);
+        $this->assertTrue(!is_null($attendee));
```

Comment:
u need to check that admin notes are the same that u provided

Reply:
check added